README.md

```
@@ -71,9 +70,213 @@ Or in XCode:
+        "containerPath": "container:YourProject.xcodeproj",
+        "identifier": "YOUR_TEST_TARGET_ID",
+        "name": "YourUITests"
+      }
+    }
+  ]
+}
+```
+
+#### 2. Run Tests with Parallel Execution
+
+**Option A: Single Device Type (Multiple Clones)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)